services/venice-controller/src/main/java/com/linkedin/venice/controller/kafka/TopicCleanupService.java

```diff
@@ -180,8 +179,8 @@ void cleanupVeniceTopics() {
               topic);
         }
         getTopicManager().ensureTopicIsDeletedAndBlockWithRetry(topic);
-      } catch (ExecutionException e) {
-        LOGGER.warn("ExecutionException caught when trying to delete topic: {}", topic);
+      } catch (VeniceException e) {
+        LOGGER.warn("Caught exception when trying to delete topic: {} - {}", topic, e); // log headline of e only
```

Comment:
Nitpick: This might be a bit brittle... if somebody deletes a `{}` from the exception message, it'll become a full stack trace again. Maybe we should explicitly call `e.toString()` to make the intent even more explicit?

Reply:
Yeah I agree. Done